src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -193,21 +191,14 @@ private void Analyze(TOptions analyzeOptions, AggregatingLogger logger)
                 ? options.Threads
                 : (Debugger.IsAttached) ? 1 : Environment.ProcessorCount;
 
-            var channelOptions = new BoundedChannelOptions(2000)
-            {
-                SingleWriter = true,
-                SingleReader = true,
-            };
-            readyToHashChannel = Channel.CreateBounded<int>(channelOptions);
-
-            channelOptions = new BoundedChannelOptions(2000)
+            var channelOptions = new BoundedChannelOptions(25000)
```

Comment:
>25000 [](http://example.com/codeflow?start=59&length=5)

We allow the scan phase to build up 25k targets, as candidates, or completed files that haven't triggered the logging thread.

This all designed to maximize CPU utilization for the scan phase.